Gemfile.lock

```diff
@@ -159,7 +159,7 @@ GEM
       slop (~> 3.4)
     pry-rails (0.3.4)
       pry (>= 0.9.10)
-    puma (2.13.4)
+    puma (3.4.0)
```

Comment:
I think this means we can undo the changes from https://github.com/18F/dolores-landingham-bot/pull/165, yeah? Also, mind dropping a link here to the issue that described the fix?

/cc @jrmils89